items.md

```diff
@@ -564,6 +564,18 @@ Return codes:
 * 403 Forbidden - if you are not logged in with sufficient permissions and versioning is not public
 * 404 Not found - if the item doesn't exist
 
+### Get Item Submitter
+**/api/core/items/<:uuid>/submitter**
+
+It returns the submitter of the item
+
+Status codes:
+* 200 OK - returning the submitter
+* 204 No Content - if there is no submitter for the specified item
```

Comment:
Note: I see a 204 here when access is denied on the EPerson findOne PreAuthorization checks. I think this is intended -- `ConverterService` returns null for `EPersonRest#findOne` when evaluating preauth check, rather than throwing a hard authorization error, and this means the ItemSubmitterLinkRepository response looks like a "No Content", not an authZ failed.
Do we just need to document this case here, or is there also a change needed in the handling?

Reply:
Done